backend/apps/github/api/v1/repository.py

```
@@ -0,0 +1,31 @@
+
+from django.http import HttpRequest
+from ninja import Router, Schema
+from ninja.errors import HttpError
+from ninja.pagination import PageNumberPagination, paginate
+
+from apps.github.models.repository import Repository
+
+router = Router()
+
+
+class RepositorySchema(Schema):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)